### DISCUSSION TITLE
Renderable mirrored on screen

### DISCUSSION THREAD

**Discussion by gsechan** [sceneview/sceneview#420] [Q&A] [answered]
I'm using ViewNode to render a view onto an arsceneview.  It's rendering, but it's mirrored-  the left is on the right and view versa.  So my TextView is backwards.  I tried applying a 180 degree rotation to each axis independently.  That did manage to move it upside down, but not flip it.  Any ideas on how to make it work?

My code for creating the node:

```
                        val node = ViewNode(sceneView.engine, sceneView.modelLoader, attachmentManager)
                       node.loadView(this@MainActivity, R.layout.marker, onLoaded = { instance, view ->
                            rootNode.addChildNode(node)
                        })

```

**Comment by gsechan** [comment, marked as answer]
Actually I can solve my own problem.  Applying node.scale = Scale(-1f, 1f, 1f) fixed it

**Comment by SyedMoizoffical** [comment]
hello every one .
we create app that have features ,measurement(Tap)
any one have code for this ,or guide me how to achieve this in sceneview
